bin/GameIndex.dbf

```diff
@@ -8620,6 +8686,11 @@ Region = PAL-G
 Serial = SLES-50848
 Name   = Speed Kings
 Region = PAL-M5
+[patches = 4c133d9d]
+	author=Prafull
+	// Fixes hang before going ingame
```

Comment:
It would be a nice idea to indicate what this patch is, in this case, this patch remove the need of the VIF FIFO.

Reply:
Yeah, im not familiar with patching yet but i will add those notes if u find more of them. (Forum doesnt give me everything)

Reply:
Thanks, as a side note, you can add this is needed because the VIF FIFO handling in PCSX2 is not accurate enough for this game.